src/styles/header.scss

```diff
@@ -0,0 +1,68 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.body {
+  font-weight: 500;
+  line-height: 14px;
+  font-size: 12px;
+  background: #fff;
+  margin: 0 auto;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+  width: 100%;
+
+  &__moyo-logo {
```

Comment:
It is better to use the class name '.header__logo' instead of '.header__moyo-logo' for better consistency in the BEM naming convention.